tests/integration/adapter-test.js

```diff
@@ -597,3 +597,37 @@ test('Saving a record with a quotes in a string attribute', function(assert) {
     });
   });
 });
+
+test('Saving a record with a back slash in a string attribute', function(assert) {
+  assert.expect(2);
+
+  run(function() {
+    store.push({
+      data: {
+        type: 'post',
+        id: '1',
+        attributes: {
+          name: 'Rails is omakase'
+        }
+      }
+    });
+  });
+
+  ajaxResponse({
+    data: {
+      post: {
+        id: '1',
+        name: '\\\\Ember.js is da bomb.'
+      }
+    }
+  });
+
+  run(function() {
+    let post = store.peekRecord('post', 1);
+
+    post.save().then(function(post) {
```

Comment:
I think you're missing the step where you set the name of the post. For example:
```javascript
post.set('name', 'Ember.js is da bomb / real deal.');
```

 That's where you'd use the character to be escaped.